### PR TITLE
[CHK-1321] fix: null paymentTypeCode bundle handling

### PR DESCRIPTION
**Pull request by MarcoAGreco**
#### List of Changes

- Add `null` value handling for bundle paymentType

#### Motivation and Context

- The `null` value is considered as ANY value in the afm domain, to embrace this logic the null value of paymentType is converted to the requested one by the client.

#### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java [hunk 1]
```diff
@@ -239,4 +240,46 @@ void shouldRetrieveFeeWithoutPspList() {
         assertEquals(gecResponse.getBundleOptions().size(), serviceResponse.getBundles().size());
     }
 
+    void shouldRetrieveFeeWithPspWithNullPaymentType() {
```
Comment (Fernando-Granato) [inline src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java:243]: The @Test annotation is missing 🙂

File: src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java [hunk 2]
```diff
@@ -239,4 +240,46 @@ void shouldRetrieveFeeWithoutPspList() {
...
+                .thenReturn(Mono.just(gecResponse));
+
+        paymentMethodService = new PaymentMethodService(afmClient, paymentMethodRepository, paymentMethodFactory);
+
+        CalculateFeeResponseDto serviceResponse = paymentMethodService
+                .computeFee(Mono.just(calculateFeeRequestDto), paymentMethodId, null).block();
+        assertEquals(paymentTypeCode, serviceResponse.getBundles().get(0).getPaymentMethod());
+    }
+
+    public void shouldGetNullPaymentMethod() {
```
Comment (Fernando-Granato) [inline src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java:275]: The @Test annotation is missing

File: src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java (hunk 1 above)
Comment (MarcoAGreco) [reply src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java:244]: Thanks 👍 solved with 89dcc77

**Comment by MarcoAGreco** [comment]
> Test coverage does not reach a minimum and reports errors
This was caused by the missing `@Test` annotation, I don't know why the sonarcloud comment was not updated, you can see the results here 
https://sonarcloud.io/summary/new_code?id=pagopa_pagopa-ecommerce-payment-methods-service&branch=CHK-1321-fix-null-paymenttype

**Review by Fernando-Granato — COMMENTED** [review]
well done 🚀

**Review by Fernando-Granato — APPROVED** [review]

**Pull request merged**